lib/matplotlib/widgets.py

```diff
@@ -1386,18 +1468,15 @@ def new_axes(self, ax):
                                        **self.rectprops)
             self.ax.add_patch(self.stay_rect)
 
-        if not self.useblit:
-            self.ax.add_patch(self.rect)
-            self.artists = [self.rect]
+        self.ax.add_patch(self.rect)
+        self.artists = [self.rect]
 
     def ignore(self, event):
         """return *True* if *event* should be ignored"""
         return _SelectorWidget.ignore(self, event) or not self.visible
 
-    def press(self, event):
+    def _press(self, event):
```

Comment:
What's the reason for this API change? While I doubt anyone is directly using these, could it mess up people would might be subclassing this Widget?